src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -1139,6 +1127,42 @@ private bool SetLayoutFromElement(object o, NLogXmlElement layoutElement)
             return false;
         }
 
+        private bool SetItemFromElement(object o, NLogXmlElement element)
+        {
+            if (element.Value != null)
+                return false;
+
+            string name = element.LocalName;
+
+            PropertyInfo propInfo;
+            if (!PropertyHelper.TryGetPropertyInfo(o, name, out propInfo))
+            {
+                return false;
+            }
+
+            object item = propInfo.GetValue(o, null);
+            this.ConfigureObjectFromAttributes(item, element, true);
+            this.ConfigureObjectFromElement(item, element);
+            return true;
+        }
+
+        private void ConfigureObjectFromAttributes(object targetObject, NLogXmlElement element, bool ignoreType)
```

Comment:
Please undo the move of the method(s) . We need to merge this also with the coreclr branch, and now it's likely to get conflicts